### PR TITLE
Fix Pip Dependencies 

### PR DESCRIPTION
**Pull request by enjeyw**
Gql just got updated from 2.0.0 to 3.0.0. This is a breaking change, as it requires requests-toolbelt to be installed.
This means a current build of Seer-py right now won't work.

This PR adds the required deps, and pins all python deps to something sensible so that this is less likely to happen in the future.

### CODE REVIEW

**Review by norrishd — DISMISSED** [review]
Thanks for the fix!

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
-gql
-requests
-numpy
-pandas
-# TODO: graphql-core 3.0.0 has a problem as at 02/12/2019
-# revisit in the future, we may be able to remove the restriction
-graphql-core<3.0.0
-pyjwt[crypto]
-tqdm
+gql==3.0.0
```
Comment (ronrest) [inline requirements.txt:1]: if GQL is the only one that is causing the issue, then perhaps this is the only one whose version should be pinned. 
I am worried that pinning pandas and numpy will cause dependency version clashes with other repos that depend on seerpy, but which use a different version of pandas and numpy.

**Review by ronrest — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 1 above)
Comment (enjeyw) [reply requirements.txt:1]: fair enough - will change!
(noting for the record that this leaves the door open for it to happen again)

Comment (norrishd) [reply requirements.txt:1]: That's a good point to think about. Do you know which repos use seer-py? Off the top of my head I can think of:

- seer-worker
- seer-risk-algo
- seer-api-events (some lambdas)
- seer-disco (probably?)
- seer-holter (probably?)

Comment (ronrest) [reply requirements.txt:1]: - I know `seer-holter` uses it. 
- I dont know what else.

Comment (enjeyw) [reply requirements.txt:1]: just done a minimal pin required to pass tests :)
lemme know what u think @ronrest

Comment (ronrest) [reply requirements.txt:1]: There used to be the following note on the requirements file:

```bash
# TODO: graphql-core 3.0.0 has a problem as at 02/12/2019
# revisit in the future, we may be able to remove the restriction
```
@norrishd , do you have any idea what the **problem** faced at that time was? And if it is covered by one of the tests? It has now been updated to `3.2.0`

Comment (norrishd) [reply requirements.txt:1]: No I'm not sure, I think basically a bug in that version. I experimentally removed that pin in `pyql` and we've been using it for months now with no problem so presumably it's long been fixed.

Comment (ronrest) [reply requirements.txt:1]: cool

**Review by ronrest — APPROVED** [review]

**Pull request merged**